drivers/vcnl40x0/Kconfig

```diff
@@ -0,0 +1,33 @@
+# Copyright (c) 2021 HAW Hamburg
+#
+# This file is subject to the terms and conditions of the GNU Lesser
+# General Public License v2.1. See the file LICENSE in the top level
+# directory for more details.
+#
+
+if TEST_KCONFIG
+
+choice
+    bool "VCNL4010/VCNL4020/VCNL4040 Proximity and Ambient Light sensors"
```

Comment:
Found it, this is missing the `optional`, so the one variant of the module was always selected